flocker/node/agents/blockdevice.py

```diff
@@ -688,6 +688,32 @@ def got_compute_id(compute_instance_id):
 
 
 @implementer(IStateChange)
+class AttachNeeded(PClass):
+    """
+    We need to attach an unattached volume to this node (the node of the
+    deployer it is run with) but lack the necessary information to do so.
+
+    Creating this is still useful insofar as it may let the convergence
```

Comment:
Well, it's better information than "too stale". If you've reached this point you know that you probably would've attached, i.e. actions would be taken. So maybe `ActionNeeded` or something.